fastlane/Fastfile

```diff
@@ -145,7 +146,20 @@ def distribute_builds(
       suffix: 'full',
       extension: 'nupkg',
       name: 'Windows Update'
-    }
+    },
+    windows_appx: {
+      binary_path: File.join(BUILDS_FOLDER, 'Studio-appx-signed', "Studio #{normalized_version}.0.appx"),
+      filename_core: 'win32',
+      extension: 'appx',
+      name: 'Windows (Appx)'
+    },
+    windows_appx_unsigned: {
+      binary_path: File.join(BUILDS_FOLDER, 'Studio-appx-unsigned', "Studio #{normalized_version}.0 unsigned.appx"),
```

Comment:
This approach doesn't work for dev builds as the build script looks for the `Studio 287a659.0.appx` file name, but the actual filename received from the previous step is `Studio 1.3.0.0.appx`.

Can we remove the version from the files to match behavior for other artifacts?